app/src/main/java/io/apicurio/registry/auth/AuthConfig.java

```diff
@@ -37,61 +38,78 @@ public class AuthConfig {
     Logger log;
 
     @ConfigProperty(name = "registry.auth.enabled", defaultValue = "false")
+    @Info( category = "auth", description = "Enable auth", availableSince = "2.0.0.Final")
     boolean authenticationEnabled;
 
```

Comment:
Suggest "Enable authentication" (applies to multiple properties, i.e., spell out `authentication` or `authorization`, whichever applies)

Reply:
👍